static/app/components/group/suggestedOwnerHovercard.tsx

```diff
@@ -213,7 +216,7 @@ const OwnershipValue = styled('code')`
   line-height: 1.2;
 `;
 
-const EmailAlert = styled(p => <Alert iconSize="16px" {...p} />)`
```

Comment:
do we lose the `iconSize` here?

Reply:
https://github.com/getsentry/sentry/blob/d2dc259e26253d98273b6287f4c68a93683c7ac4/static/app/components/alert.tsx#L87-L105

there is no icon size prop on this